src/applications/facility-locator/utils/facilityAddress.js

```diff
@@ -1,15 +1,16 @@
 import { compact, isEmpty } from 'lodash';
 import { LocationType } from '../constants';
+import { titleCase } from 'applications/vaos/utils/formatters';
```

Comment:
It would be better for you to copy this code into the facility-locator code, or if you think it's broadly useful, add it somewhere in the platform folder, instead of using it from vaos directly. Keeping applications as independent as we can helps avoid teams accidentally breaking other teams code without knowing it and also allows us to refactor our own apps without needing to change someone else's.

Reply:
Yeah, I was wondering about that.

I did create a Facility-specific version of the `titleCase` function, but it has some domain-specific behavior that we don't need in all cases. I'll modify our function to be more flexible, and use it instead.